analyze_logs.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex # -e: exit early on any error. -x: write all commands to stderr
 
+date '+%Y-%m-%d'
```

Comment:
should the date be written to canary_tests.log? how will the monitor look for it here?

Reply:
The monitor makes sure the analysis has run on each machine. The analysis on a machine (already) makes sure the tests have run on that machine. (To get the latter to work, we're already printing this line in the tests.)

That way, if there's some crazy situation in which the tests run but the analysis doesn't (and therefore we wouldn't find out in Slack if the tests failed), the monitor will notice and report to us.

So, the topology looks like this, where machine B is the monitor:

machine A: run tests -> run analysis \
machine B: run tests -> run analysis -- monitor that analysis has run on all machines
machine C: run tests -> run analysis /